json_to_avro/schema_provider.py

```diff
@@ -0,0 +1,59 @@
+from schema_registry.client import AsyncSchemaRegistryClient, SchemaRegistryClient
+from schema_registry.client.utils import SchemaVersion
+
+from json_to_avro.avro_schema.avro_schema_candidate import AvroSchemaCandidate
+from json_to_avro.avro_schema.registered_avro_schema import RegisteredAvroSchema, RegisteredAvroSchemaId
+
+
+class SchemaProvider:
```

Comment:
I keep coming back to this class. It bothers me, but I think that _not_ having it bothers me more. 🤷 
I really really dislike the schema registry implementations...